package.json

```diff
@@ -2,61 +2,75 @@
   "name": "bind-event-listener",
   "version": "3.0.0",
   "private": false,
-  "description": "Making binding and unbinding DOM events easier",
+  "description": "Safe DOM Event binding and unbinding",
   "author": "Alex Reardon <alexreardon@gmail.com>",
   "license": "MIT",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/alexreardon/bind-event-listener.git"
-  },
   "keywords": [
+    "event listener",
+    "event handler",
+    "dom events",
     "dom",
     "ui",
-    "event listener",
-    "event handler"
+    "bind"
   ],
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/alexreardon/bind-event-listener.git"
+  },
   "sideEffects": false,
+  "type": "module",
```

Comment:
@Andarist I have gone for the approach outlined in the [Node conditional exports](https://nodejs.org/api/packages.html#conditional-exports) documentation.

```json
// package.json
{
  "exports": {
    "import": "./index-module.js",
    "require": "./index-require.cjs"
  },
  "type": "module"
} 
```

So it's a listed as a "module", but the when the "require" conditional export is being used, we force the output to be commonjs by using the `.cjs` file extension

Reply:
I am doing some testing now, but this seems really promising